ergo-wallet/src/test/scala/org/ergoplatform/wallet/utils/Generators.scala

```diff
@@ -173,4 +184,32 @@ trait Generators {
   } yield TrackedBox(
     creationTxId, creationOutIndex, inclusionHeightOpt, spendingTxIdOpt, spendingHeightOpt, box, appStatuses)
 
+
+  def unsignedTxGen(secret: SecretKey): Gen[(IndexedSeq[ErgoBox], UnsignedErgoLikeTransaction)] = {
```

Comment:
Seems like way too simple and fixed tx contents. Given it's generic name I suppose it'd be used everywhere. Consider extending the bounds (dynamic number of inputs and outputs, non-empty regs, tokens, etc.).